mkenv.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import importlib
+import os
+import subprocess
+import sys
+
+# Requires at least Python 3.7. If placing this into a project that
+# requires a newer Python, you may elect to increase the version
+# requirement below.
+if sys.version_info[:2] < (3, 7):
+    raise SystemError('Only Python 3.7+ is allowed')
+
+MKENV_IMPL = 'mkenv_impl'
+HERE = os.path.dirname(os.path.abspath(__file__))
+MKENV_SUBDIR = '.mkenv'
+MKENV_REPO = os.environ.get('DL_MKENV_REPO', 'git@octocat.dlogics.com:datalogics/mkenv.git')
```

Comment:
We've been changing all our `mkenv.py` files to refer to the mkenv project in the cloud: `git@github.com:datalogics/mkenv.git`

Reply:
Added